api/src/main/java/org/openmrs/api/db/hibernate/HibernateSessionFactoryBean.java

```diff
@@ -56,6 +57,9 @@ public class HibernateSessionFactoryBean extends LocalSessionFactoryBean impleme
 	// This will be sorted on keys before being used
 	@Autowired(required = false)
 	public Map<String, Interceptor> interceptors = new HashMap<>();
+
+	@Value("${cache_type:local}")
+	private String cacheType;
 	
 	private Metadata metadata;
 	
```

Comment:
Should we also remove this? https://github.com/openmrs/openmrs-core/blob/f08e088d16be72d5332ff8699dde57177ff4ff96/api/src/main/java/org/openmrs/api/db/hibernate/HibernateSessionFactoryBean.java#L211-L212

Reply:
And this? https://github.com/openmrs/openmrs-core/blob/master/webapp/src/main/resources/log4j2.xml#L46